### ISSUE TITLE
How to add ETF Symbols and/or Yahoo Symbols?

### ISSUE THREAD

**Issue by BlackArbsCEO** [camisatx/pySecMaster#4]
First, thanks for your work here, I like the structure and utility of the database. However I'm fairly inexperienced around DB management, and I'm unclear on how to add new symbols to the DB and how that works with your custom TSID symbology. I assume it's much simpler than I think but would really appreciate some guidance. Thanks.

**Comment by camisatx** [comment]
Hey BlackArbsCEO, thanks for viewing this project and the kind words. I went ahead and improved the documentation of the readme to clarify more about what is going on under the hood of this system.

In regards to ETF data, the best I can recommend right now is to use [Quandl's Google Finance database](https://www.quandl.com/data/GOOG) with ETFs selected in the GUI (See step 6 of the updated ['Quandl daily data'](https://github.com/camisatx/pySecMaster#quandl-daily-data) Quick Start Guide; if you select **'quandl_goog_etf'** for the _Quandl Data_ combo-box in the _Data_ tab, Google finance daily data will be downloaded for all ETFs).

For Yahoo data specifically, I don't have a price extractor built to pull it directly from them. The data could either be pulled from Quandl's Yahoo database, or it could be downloaded straight from Yahoo. I think the later should be easier to make. I've added it to my list of things to do.

Let me know if you have any other questions or ideas!
